NEWS.md

```diff
@@ -67,3 +67,6 @@
   ([#2315](https://github.com/JuliaData/DataFrames.jl/pull/2315))
 * add rich display support for Markdown cell entries in HTML and LaTeX
   ([#2346](https://github.com/JuliaData/DataFrames.jl/pull/2346))
+* limit display width in `text/plain` to approximately 32 screen characters
```

Comment:
Is it really approximate? AFAICT, you stop at 32 and take the previous character, so it's always the maximum width before going beyond 32, right?

Reply:
No, because some characters have more than `1` text width (in which case we can get a bit different result)

Reply:
Yes but since you return the last character before going beyond 32, the result will always be less than 32, right? I think it would be worth mentioning in the docstring rather than just saying it's approximate.

Reply:
The precise statement is that the result without `…` has maximal text width less or equal than 32 display characters, but how would you phrase it so that this would be readable?

Reply:
Maybe something like this? Anyway that's internal, what matters is that it's correct even if it's not super easy to read. :-)
```
`truncstring` indicates the maximal width the output can use before being truncated
(in the `textwidth` sense, excluding `…`).
```

Reply:
OK - I have proposed something along these lines :smile: